src/utils/components/SSHSecretSection/components/SSHKeyUpload/SSHKeyUpload.tsx

```diff
@@ -26,6 +27,7 @@ type SSHKeyUploadProps = {
 
 const SSHKeyUpload: FC<SSHKeyUploadProps> = ({ secrets, setSSHDetails, sshDetails }) => {
   const { t } = useKubevirtTranslation();
+  const [activeNamespace] = useActiveNamespace();
```

Comment:
is there a chance that this component is used in a `all-namespaces` page?
In that case, `activeNamespace` will have the value of `ALL_NAMESPACES_SESSION_KEY` and not `undefined`

Reply:
Yes, there is. I fixed it to account for that. Thanks!